fuzzamoto-ir/src/operation.rs

```diff
@@ -137,6 +149,29 @@ impl fmt::Display for Operation {
                 write!(f, "LoadConnectionType(\"{}\")", connection_type)
             }
             Operation::LoadDuration(duration) => write!(f, "LoadDuration({})", duration.as_secs()),
+            Operation::LoadAddr(addr) => match addr {
+                AddrRecord::V1 {
+                    time,
+                    services,
+                    port,
+                    ..
+                } => write!(
+                    f,
+                    "LoadAddr(time={}, services={}, port={})",
+                    time, services, port
+                ),
+                AddrRecord::V2 {
+                    time,
+                    services,
+                    network,
+                    port,
+                    ..
+                } => write!(
+                    f,
+                    "LoadAddrV2(time={}, services={}, network={}, port={})",
```

Comment:
```suggestion
                    "LoadAddr({}, {}, {})",
                    time, services, port
                ),
                AddrRecord::V2 {
                    time,
                    services,
                    network,
                    port,
                    ..
                } => write!(
                    f,
                    "LoadAddrV2({}, {}, {}, {})",
```

nit: it doesn't really matter but the other load operations are also not displayed with param names